guideGlobalMetadata.json

```
@@ -1,11 +1,12 @@
 {
     "supported_interfaces": [
         "SpikeGLXRecordingInterface",
+        "SpikeGLXNIDQInterface",
         "PhySortingInterface",
         "NeuroScopeRecordingInterface",
+        "NeuroScopeLFPInterface",
         "NeuroScopeSortingInterface",
         "BiocamRecordingInterface",
         "IntanRecordingInterface",
```

```suggestion
        "IntanRecordingInterface"
```
re-arranged interface to match the names closer but botched the merge conflict